### ISSUE TITLE
Add code of conduct to midl.io

### ISSUE THREAD

**Issue by tvercaut** [MIDL-Conference/midl-website#9]
We could just reuse the one from 2019: https://2019.midl.io/code-of-conduct/

**Comment by nlessmann** [comment]
I've added a page with the same text, but haven't placed a link to it anywhere yet. Not sure where best to do that.

https://www.midl.io/code-of-conduct.html

**Comment by bramvanginneken** [comment]
I reopen it because I think we should look at the text. I compared it with the ICLR code at https://iclr.cc/public/CodeOfConduct and I think that one is a bit clearer, it does not use very vague phrases like ‘disruptive conduct’.

**Comment by bramvanginneken** [comment]
This paper has useful phrases to make a better code of conduct: https://www.frontiersin.org/articles/10.3389/fmars.2016.00103/full
Eg, instead of disruptive conduct it says something that everybody understands: ‘Disrespectful disruption of presentations.’

**Comment by bramvanginneken** [comment]
Another link: https://confcodeofconduct.com/

**Comment by nlessmann** [comment]
Maybe this should be picked up again? The 2020 code of conduct is now different from the one on midl.io, see https://2020.midl.io/code-of-conduct.html

**Comment by tvercaut** [comment]
Yes indeed. This is a point to discuss with the board. Here ia a relevant exchange with Tal:
